libs/base-utils/project.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "base-utils",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/base-utils/src",
```

Since this is shared, I prefer `libs/utils`, and maybe making this the `base` export.
```suggestion
  "sourceRoot": "libs/utils/src",
```